NEWS

```diff
@@ -1,3 +1,25 @@
+26-Apr-2019 Ignition 2.0.0-beta
+
+  Features:
+    - Add configuration spec 3.1.0-experimental
+    - Add support for proxies in spec 3.1.0-experimental
```

Comment:
Maybe "Allow specifying network proxies in..."?

Reply:
How about "Allow specifying HTTP(S) proxies in...."

Reply:
Sure.